build/check_binary_size.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
```

Comment:
This file and bootstrap.sh are dupliated in build/scripts. Merge issue?

Reply:
Yea these files must have been retained in the previous merge, but they should be deleted now.. I'll remove in this PR